tardis/io/configuration/config_validator.py

```diff
@@ -83,11 +83,13 @@ def is_quantity(checker, instance):
 
 
 def validate_dict(
-    config_dict, schemapath=config_schema_file, validator=DefaultDraft4Validator
+    config_dict,
+    schemapath=CONFIG_SCHEMA_FNAME,
+    validator=DefaultDraft4Validator,
 ):
     with open(schemapath) as f:
         schema = yaml.load(f, Loader=YAMLLoader)
-    schemaurl = "file://" + schemapath
+    schemaurl = f"file://{schemapath}"
```

Comment:
I'm not sure if the format string is actually more readable than the concatenation here.

Reply:
I'm not a fan of string additions and thing this is much cleaner - unless you strongly disagree

Reply:
Is there a lot of difference between the two methods??

Reply:
f-strings do automatic string conversion. I think also we try to use those more in python. I think it's a stylistic choice mostly.